src/lib/utils/thorchain/hooks/useSendThorTx.tsx

```diff
@@ -73,6 +76,8 @@ type UseSendThorTxProps = {
   dustAmountCryptoBaseUnit?: string
 }
 
+const actionsWithDustAmount: Action[] = ['withdrawLiquidity', 'withdrawSavers', 'claimTcy']
```

Comment:
Extracted and added a type so we can have strong typing on the predicate

Reply:
Also for anyone reading this, only claim has dust amount:
https://stagenet-thornode.ninerealms.com/thorchain/tx/details/d1e4056a7456964a9176428d83e12010f5f577986ebb8001df7edb6115bb7f10

Stake does not as it's a THORChain Tx with THOR.TCY value (we'll need to handle that in THOR adapter, as we only ever sent THOR there before): https://stagenet-thornode.ninerealms.com/thorchain/tx/details/FFDF74263B564BC1BE0812BF1E3A16E8652DC395251B549D776E9810BA30B897
And ditto for unstake, it's a good ol' unstake Tx with 0 RUNE value: https://stagenet-thornode.ninerealms.com/thorchain/tx/details/AE10B40187A79914D7E7EE59F109B8B10C6EF33D1790F4570F9FE44504D4F3F4